src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -12,7 +12,7 @@
   <Import Project="..\..\build\Versioning.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
We might need to add .net 9 SDK for the CICD pipeline

Reply:
https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/build/install-dotnet.ps1